core/src/main/java/com/sequenceiq/cloudbreak/core/flow2/externaldatabase/provision/handler/CreateExternalDatabaseHandler.java

```diff
@@ -80,9 +80,9 @@ public void accept(Event<CreateExternalDatabaseRequest> createExternalDatabaseRe
                         stack.getName(), stack.getStatus().name(), DetailedStackStatus.PROVISION_REQUESTED.name());
                 stackUpdaterService.updateStatus(stack.getId(), DetailedStackStatus.PROVISION_REQUESTED,
                         ResourceEvent.CLUSTER_EXTERNAL_DATABASE_CREATION_FINISHED, "External database creation finished");
-                result = new CreateExternalDatabaseResult(stack.getId(), EXTERNAL_DATABASE_WAIT_SUCCESS_EVENT.event(),
-                        stack.getName(), stack.getCluster().getDatabaseServerCrn());
+                resourceCrn = stack.getCluster().getDatabaseServerCrn();
             }
+            result = new CreateExternalDatabaseResult(stack.getId(), EXTERNAL_DATABASE_WAIT_SUCCESS_EVENT.event(), stack.getName(), resourceCrn);
```

Comment:
The resourceCrn can be null, is it ok?

Reply:
no, it's not ok, why would id be null?

Reply:
it is just a minor refactor, the functionality did not change